its/src/test/java/com/sonar/maven/it/suite/MavenTest.java

```diff
@@ -55,14 +54,9 @@
   @Rule
   public TemporaryFolder temp = new TemporaryFolder();
 
-  @Before
-  public void deleteData() {
-    orchestrator.resetData();
```

Comment:
I'm surprised that you don't need to do some clean up of projects. 
Maybe it would be safer to do these cleanup, for instance like what was done here => https://github.com/SonarSource/sonar-dotnet/blob/98c0671a0b71ec36f1fc7a237aa51a7b6e5f57d7/its/src/test/java/com/sonar/it/shared/TestUtils.java#L187